### PR TITLE
docker-machine-nfs 0.2.0 (new formula)

### PR DESCRIPTION
**Pull request by hakamadare**
Initial commit of docker-machine-nfs; this is a script that eases the
process of enabling NFS sharing (instead of the default VirtualBox) for
boot2docker-based Docker VMs.

### CODE REVIEW

File: Library/Formula/docker-machine-nfs.rb [hunk 1]
```diff
@@ -0,0 +1,16 @@
+class DockerMachineNfs < Formula
+  desc "Activates NFS on docker-machine"
+  homepage "https://github.com/adlogix/docker-machine-nfs"
+  url "https://raw.githubusercontent.com/adlogix/docker-machine-nfs/0.1.0/docker-machine-nfs.sh"
+  version "0.1.0"
+  sha256 "f7c1742d4644353bb1eb2c2ac4798fd037f7848129ae5e3add1630071521895d"
+
+  def install
+    mv "docker-machine-nfs.sh", "docker-machine-nfs"
+    bin.install "docker-machine-nfs"
```
Comment (bfontaine) [inline Library/Formula/docker-machine-nfs.rb]: You can merge both lines into `bin.install "docker-machine-nfs.sh" => "docker-machine-nfs"`.

File: Library/Formula/docker-machine-nfs.rb [hunk 2]
```diff
@@ -0,0 +1,16 @@
+class DockerMachineNfs < Formula
+  desc "Activates NFS on docker-machine"
+  homepage "https://github.com/adlogix/docker-machine-nfs"
+  url "https://raw.githubusercontent.com/adlogix/docker-machine-nfs/0.1.0/docker-machine-nfs.sh"
```
Comment (bfontaine) [inline Library/Formula/docker-machine-nfs.rb]: It’s probably better to use the tarball here so it (automatically) installs the README as well.

**Comment by hakamadare** [comment]
@bfontaine thanks for the feedback, i've made the changes you requested.

**Comment by hakamadare** [comment]
i'm happy to squash down these commits if you'd prefer.

File: Library/Formula/docker-machine-nfs.rb [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+  homepage "https://github.com/adlogix/docker-machine-nfs"
+  url "https://github.com/adlogix/docker-machine-nfs/archive/0.2.0.tar.gz"
+  sha256 "ebc023657c044e706c8d9bc275d6b96de9d1f5178c7f6a90a8ce3bc167d1ec74"
+
+  def install
+    bin.install "docker-machine-nfs.sh" => "docker-machine-nfs"
+  end
+
+  test do
+    system "#{bin}/docker-machine-nfs"
```
Comment (bfontaine) [inline Library/Formula/docker-machine-nfs.rb]: Can’t we write a test that does something more substantial than just printing the help?

Comment (hakamadare) [inline Library/Formula/docker-machine-nfs.rb]: @bfontaine the script doesn't have a no-op mode, so any testing would involve making changes to the user's system.  would printing the help and checking to make sure that a particular string matches the output be preferable?

Comment (bfontaine) [inline Library/Formula/docker-machine-nfs.rb]: Thanks. Let’s leave it as is then.

**Comment by bfontaine** [comment]
Thank you for your contribution to Homebrew, @hakamadare; we appreciate it! :tada:

**Pull request closed without merging**